src/chain_spec.rs

```diff
@@ -88,22 +102,24 @@ impl Alternative {
 				ChainSpec::from_genesis(
 					"KILT Testnet",
 					"kilt_testnet",
+					ChainType::Live,
 					|| {
 						testnet_genesis(
 							vec![
-					x25519::Public::from_raw(hex!("58d3bb9e9dd245f3dec8d8fab7b97578c00a10cf3ca9d224caaa46456f91c46c")),
-					x25519::Public::from_raw(hex!("d660b4470a954ecc99496d4e4b012ee9acac3979e403967ef09de20da9bdeb28")),
-					x25519::Public::from_raw(hex!("2ecb6a4ce4d9bc0faab70441f20603fcd443d6d866e97c9e238a2fb3e982ae2f")),
-				],
+							get_authority_keys_from_seed("0x58d3bb9e9dd245f3dec8d8fab7b97578c00a10cf3ca9d224caaa46456f91c46c"),
```

Comment:
This is supposed to be a public key

Reply:
fixed in 8d75804

Reply:
Have a look at how polkadot is doing it: https://github.com/paritytech/polkadot/blob/848e1a892f99c16015b75117fdae00b393cde770/service/src/chain_spec.rs#L365
Can't we do it similarly?